app/views/users/show.html.erb

```diff
@@ -1,33 +1,31 @@
-<div class="container prof">
-	<div class="row">
-		<div class="col-md-12">
-			  <h2 class="text-center"><%= @user.userName %> </h2>
-			  <%= image_tag @user.avatar.url, size: "320x426", class: "img-responsive" %>
-			  <h3><%=@user.first_name%> <%=@user.last_name%></h3>
-			  <% if current_user == @user %>
-			  <%= button_to "Edit", edit_user_path, method: :get, class: "btn btn-default" %>
+<div class="container">
+  <div class="row">
+    <h1 class="text-center"><%= @user.userName %> </h1>
```

Comment:
hmm... that doesn't look like a ruby name, does your user model really have a `userName` field?  If so please make a migration to change it to `name`.